internal/manager/run.go

```diff
@@ -173,13 +179,23 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 	// See https://github.com/kubernetes-sigs/kubebuilder/issues/932
 	// +kubebuilder:scaffold:builder
 
-	setupLog.Info("Starting health check servers")
-	if err := mgr.AddHealthzCheck("health", healthz.Ping); err != nil {
-		return fmt.Errorf("unable to setup healthz: %w", err)
-	}
-	if err := mgr.AddReadyzCheck("check", readyzHandler(mgr, synchronizer)); err != nil {
-		return fmt.Errorf("unable to setup readyz: %w", err)
-	}
+	// use standalone health check server instead of servers inside manager
+	// because manager depends on initial kong clients:
+	// https://github.com/Kong/kubernetes-ingress-controller/issues/3592
+	// After we implement the feature that manager does not depend on initial kong clients,
+	// we should move back to the health check server inside the manager.
+	/*
+		setupLog.Info("Starting health check servers")
+		if err := mgr.AddHealthzCheck("health", healthz.Ping); err != nil {
+			return fmt.Errorf("unable to setup healthz: %w", err)
+		}
+		if err := mgr.AddReadyzCheck("check", readyzHandler(mgr, synchronizer)); err != nil {
+			return fmt.Errorf("unable to setup readyz: %w", err)
+		}
+	*/
```

Comment:
As we already have the health server's comment explaining the problem, I think we can eliminate this snippet with comments.
```suggestion
```

Reply:
Do you mean remove all the part?

Reply:
Yup, I don't think leaving the commented code has any value as we can always get it from git.

Reply:
done